draft-ecahc-moderation.md

```diff
@@ -218,27 +231,40 @@ their own judgment and community suggestions. (Note that some participation
 channels, such as the examples given in the previous sentence, have no
 chairs or other community leadership, so the moderation team MUST handle those.)
 
+### Non-IETF communication channels and private communications excluded
+
 It is important to note that the moderation team only
 moderates *public* IETF participation channels. Their mandate does
 not extend to problematic behavior in private channels, such as
 private chat channels, direct messages, or conversations or other
 interactions outside of meetings. In such cases, the Ombudsteam
 should be approached.
 
+### IETF Working Groups
+
 The management and moderation of participation channels associated
-with various IETF groups, inculding their mailing lists, chat
-channels and in-person, remote, or interim meetings remains primarily a
-task of the relevant group's management, such as WG chairs. However,
-moderators are available for consultation and
-assistance should issues arise, and they MAY proactively confer with the group
-management over potential patterns of behavior. Group participants MAY
-and chairs SHOULD alert the moderators to problematic behavior. When moderators
+with various IETF groups, including their mailing lists, chat channels
+and in-person, remote, or interim meetings remains primarily a task of
+the relevant group's management, such as WG chairs. However,
+moderators are available for consultation and assistance should issues
+arise, and they MAY proactively confer with the group management over
+potential patterns of behavior. Group participants MAY and chairs
+SHOULD alert the moderators to problematic behavior. When moderators
 observe or are alerted to problematic behavior on such channels, they
 SHOULD consult with the group's management to jointly determine an
-appropriate moderation response. The moderation team should respect
-the views of the group management in such cases, and the group
-management should respect the moderation team's task of upholding an
-overall IETF-wide uniformity for moderation.
+appropriate moderation response.
+
+The moderation team MAY take actions against someone in a working
+group setting, but they MUST inform management of relevant groups,
+including WG chairs and area directors, when they do so.  Working
+group chairs MAY overrule those actions in their respective groups, if
```

Comment:
I don't think that a WG chair should be able to override an IETF wide moderation decision.  E.g., if the moderation teams decides to ban someone from posting to any IETF list for 5 days, then I don't think that a WG chair should be able to say "except they can post to mine".  However, I also think that this is likely to be a non-issue as in, I doubt that a WG chair would choose to do this anyway.  But stating the ultimate authority is probably still useful.

Reply:
This needs more commentary on list.

Reply:
See last commit to address mailing list discussion.  @larseggert I may have biffed the rebase; could you eye it over?

Reply:
Ok, please review.